edfio/edf.py

```diff
@@ -871,7 +879,9 @@ def slice_between_seconds(
             else:
                 start_index = start * signal.sampling_frequency
                 stop_index = stop * signal.sampling_frequency
-                signal._digital = signal._digital[int(start_index) : int(stop_index)]
+                signal._digital = signal.digital_data[
```

Comment:
I'd like these attribute names to be consistent, my preference would be `[_]digital`, but I'm fine with `[_]digital_data` too